console_backend/src/common_constants.rs

```diff
@@ -273,8 +273,8 @@ pub enum Keys {
     NEW_INS_CONFIRMATON,
     #[strum(serialize = "ANTENNA_STATUS")]
     ANTENNA_STATUS,
-    #[strum(serialize = "RECORDING_DURATION_SEC")]
-    RECORDING_DURATION_SEC,
+    #[strum(serialize = "RECORDING_START_TIME")]
+    RECORDING_START_TIME,
```

Comment:
hard to know if this breaking change is safe.

Reply:
oops yea good catch, (this one isn't related to this PR), but im pretty sure common_constants are autogenerated from python 

which is [here](https://github.com/swift-nav/swift-toolbox/blob/b9b8d3e43cd8c391be31a00296b311756377a799/swiftnav_console/constants.py#L151)

which should be OK, since we know explicitly where these capnproto packets are sent and handled